app/components/Chatbot/ChatEntry.tsx

```diff
@@ -15,8 +15,6 @@ import StampyIcon from '~/components/icons-generated/Stampy'
 import PersonInCircleIcon from '~/components/icons-generated/PersonInCircle'
 import IconStampySmall from '~/components/icons-generated/StampySmall'
 import QuestionMarkIcon from '~/components/icons-generated/QuestionMark'
-import useIsMobile from '~/hooks/isMobile'
-import {togglePopup} from '../popups'
```

Comment:
It seems like this code might have been used in mobile views. I wasn't able to test the code and see the pop-up, but I think that in principle this code was in use so this is a change to the UX.

Reply:
I wasn't able to get citation popups to work on mobile and if you aren't either then on balance I think it's okay to remove this code.